src/docs/sdk/envelopes.mdx

```diff
@@ -600,6 +600,7 @@ These limits are subject to future change and defined currently as:
 - *1MB* for event and transaction Items
 - *100 sessions* per Envelope
 - *100 pre-aggregated session buckets* per each `"sessions"` Item
+- *100KB* for monitor Check-ins
```

Comment:
nit: is lowercase 'monitor' and uppercase 'Check-in' the spelling we want?

Reply:
i thought it was odd but i'm just going off of the convention in the bullet points above